reactioncommerce/reaction
# [WIP] Stripe Connect Capture charge

Resolves #2765 

Also makes Approvals happen and be by shop for each order.

**Note** This adds filtering by shop for an order. This feels a little bit not great but seems to work. Take a close look at the `reactiveAggregate.js` file that allows us to treat an aggregated result like a collection. If it **does** work then we can use it everywhere we need to filter collections by shop.


### To Test

1. Place a multi-shop order
1. Go to the orders panel for either shop
1. Approve the order
1. Open the other shop in incognito
1. Observe the same order for the second shop is still not approved
1. Select "capture" in the first shop
1. Observe amount is captured and order status in other shop remains the same
